apps/snitch_api/lib/snitch_api_web/router.ex

```diff
@@ -21,6 +21,9 @@ defmodule SnitchApiWeb.Router do
     get("/variants/favorites", VariantController, :favorite_variants)
     resources("/products", ProductController, except: [:new, :edit], param: "product_slug")
     get("/orders/:order_number", OrderController, :fetch_guest_order)
+    post("/hosted-payment/payubiz-request", HostedPaymentController, :payubiz_request_url)
```

Comment:
why not `/hosted-payment/source`

Reply:
The approach for sending data to gateway from backend (after creating a form) is at present only for payubiz. We were doing this since there was some CORS issue associated with using the frontend for making the request. We don't know at present if other gateways will behave that way so I have kept this route only for Payubiz and haven't made it generic.
We can make it generic later on if we face the same issue.